docs/implementation-guide/sizing.md

```diff
@@ -1,101 +1,98 @@
+- Only the [generic requirements](#generic-requirements-all-node-types) apply to Compute-only nodes
+- CPU and Memory should be architected for workloads running on these nodes
 
-### Networking
-- 2 x 25GbE NICs per server for storage, application server, and workload migrations.
-- 2 x 10GbE (LACP) NIC per server for UI access and outgoing guest traffic.
-- Redundant network switches
+!!! note "Additional Resources for Workloads"
+    Remember to account for additional resources needed by your specific workloads, virtual machines, and applications when planning your deployment.
 
 ---
-## Maximum and Minimum Supported Hardware Specifications[^1]
-
-| Resource                        | Minimum | Maximum | Resource Type |
-|---------------------------------|---------|---------|---------------|
-| nodes per system (physical servers) |   1 | 65000   | node          |
-| VMs per host                    |         |         | vm            |
-| Disks per VM (virtio SCSI)[^3]  |         | 2000    | vdisk         |
-| Storage per system              |         |         | terabyte      |
-| Storage per node                |         |         | terabyte      |
-| Individual physical disk size   |         | 64      | terabyte      |
-| vDisk size                      |         | 2000    | terabyte      |
-| Logical processors per host     |         |         | procs         |
-| RAM per host[^2]                    |         | 5       | terabyte      |
-| RAM per VM                      |         |         | terabyte      |
-| Virtual processors per host     |         |         | procs         |
-| Virtual processors per VM       |         |         | procs         |
-| clusters per system             |         | 1000    | cluster       |
-| tiers of storage per system     |         | 5       | tiers         |
-| storage clusters per system?    |         |         | cluster       |
-| failure domains per system      |         | 2       |               |
-| nodes per cluster               |         | 256     | node          |
-
-
-[^1]: As of v4.12
+
+## Maximum Supported Hardware Specifications
+
+The following table outlines the maximum supported hardware specifications for various resources in the VergeOS system as of version 4.12:
+
+| Resource                        | Maximum | Resource Type |
+|---------------------------------|---------|---------------|
+| Nodes per system                | 200     | node          |
+| Disks per VM [^3]               | 2000    | vdisk         |
+| Individual physical disk size   | 64      | terabyte      |
+| vDisk size                      | 2000    | terabyte      |
+| RAM per hos [^2]                | 5       | terabyte      |
+| Clusters per system             | 1000    | cluster       |
+| Tiers of storage per system     | 5       | tiers         |
+| vSAN Fault domains per system   | 2       | vSAN          |
+| Systems w/o Dedicated Controller Nodes | 10 | node |
```

Comment:
this is not clear, are we saying the max nodes in a system when the controllers participate in the vsan is 10?